pom.xml

```diff
@@ -12,7 +12,7 @@
 
     <groupId>org.jboss.aerogear</groupId>
     <artifactId>digger-java-client</artifactId>
-    <version>1.0.0-SNAPSHOT</version>
+    <version>1.0.2-SNAPSHOT</version>
```

Comment:
why was this done ? 

Usually this is all handled by proper tooling (e.g. the maven release plugin).

Manually bumping versions is an extremely bad practice!